src/main/java/org/rumbledb/expressions/flowr/FlworVarDecl.java

```diff
@@ -31,7 +31,7 @@
 import java.util.ArrayList;
 import java.util.List;
 
-public abstract class FlworVarDecl extends FlworClause {
+public abstract class FlworVarDecl extends Clause {
```

Comment:
Same idea with using full variable names: shall we refactor this to FlworVariableDeclaration?

Reply:
I have not refactored that part yet, but this is on the agenda as well.